config/config_test.go

```diff
@@ -41,7 +42,15 @@ const (
       Cache-Control: public, s-maxage=600, max-age=600
     AdditionalResponseHeaders:
       Access-Control-Allow-Methods: GET, POST, OPTIONS
-  
+    Transports:
+      - Name: DefaultTransport
```

Comment:
I think that a name `DefaultTransport` is misleading, as it suggests that the transport supports all kinds of request, when in fact, it doesn't support a `Head` request.